sites/docs/pages/components/date-range.md

```diff
@@ -138,6 +149,19 @@ A manually specified end date to use for the range
 Title to display in the Date Range component
 
 </PropListing>
+
+
+<PropListing 
+    name="omitGroup"
+    options="{["days", "months", "last", "todate"]}"
```

Comment:
I'm not sure this is quite the right UX.
1. I agree that the names are not quite right in lowercase. Either should be String or camelCase.
2. I think this should probably be opt in rather than opt out, if you specify anything.
3. There needs to be a mechanism that allows you to pick within groups. You might want to include last 30 days, but not last 7 days, for example.

something like

<DateRange
   name=name_of_date_range
   presets={["Last 7 Days", "Last 30 Days", "Last 3 Months", "Last Year"]}
   ...
/>